examples/run_nutation.py

```diff
@@ -82,16 +82,17 @@
         SW_kHz=config_dict["SW_kHz"],
         ret_data=data,
     )
-data.setaxis("indirect", p90_range_us * 1e-6).set_units("indirect", "s")
+data.rename("indirect", "p_90")
+data.setaxis("p_90", p90_range_us * 1e-6).set_units("p_90", "s")
 # {{{ chunk and save data
-data.chunk("t", ["ph1", "t2"], [4, -1])
-data.setaxis("ph1", ph1_cyc / 4)
+data.chunk("t", ["ph2", "ph1", "t2"], [2, 2, -1])
```

Comment:
Is this correct? Do you list the phase cycle in reverse order?  I am asking because I do not know.... Would need to look at another example and I'm doing this on my phone

Reply:
I am basing this off of how the IR is chunked in combined_ODNP which uses ph1_cyc and ph2_cyc and seems to be chunked correctly